spec/game_spec.rb

```diff
@@ -13,27 +13,53 @@ def output(message)
 end
 
 class TestPresenter
-  def print_board
-    "______"
+  def print_board(board)
+    board.board
   end
```

Comment:
board.board sounds odd.  Is there a better name than board for the instance variable? I'm not good with naming.

Reply:
Perhaps `spaces` or `cells`?

Reply:
I like the sound of cells. Renamed.